docs/conflicts.md

```
@@ -33,24 +33,46 @@ conflicts with the following logic:
+Under the hood, `spock.delta_apply()` writes a row into `pg_seclabel`
+with `provider = 'spock'` and `label = 'spock.delta_apply'`.  The
+binary-upgrade compatibility shim that translates legacy spock 5.x
+reloptions during `pg_upgrade` writes the same canonical label, so
+operators can audit the catalog uniformly:
+
+```sql
+SELECT * FROM pg_seclabel
+ WHERE provider = 'spock' AND label = 'spock.delta_apply';
+```
+
+#### Upgrading from spock 5.x
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)